api/v1/views/__init__.py

```diff
@@ -1,5 +1,7 @@
 from flask import Blueprint
-from api.v1.views.index import *
 
 # Variable app_views which is an instance of Blueprint
 app_views = Blueprint('app_views', __name__, url_prefix='/api/v1')
+
+
+from api.v1.views.index import *
```

Comment:
in this case, the import must be on the bottom cause, the API never into the index.py